infra/cifuzz/fuzz_target_test.py

```diff
@@ -87,5 +94,113 @@ def test_with_invalid_error_string(self):
     self.assertIsNone(self.test_target.get_test_case(' Example crash string.'))
 
 
+class CheckReproducibilityAndRegressionUnitTest(unittest.TestCase):
+  """Test check_reproducibility_and_regression function fuzz_target module."""
+
+  def setUp(self):
+    """Sets up dummy fuzz target to test is_reproducible method."""
+    self.test_target = fuzz_target.FuzzTarget('/example/do_stuff_fuzzer', 10,
+                                              '/example/outdir', 'example')
+
+  def test_with_valid_crash(self):
+    """Checks to make sure a valid crash returns true."""
+    with unittest.mock.patch.object(
+        fuzz_target.FuzzTarget, 'is_reproducible',
+        side_effect=[True, False]), tempfile.TemporaryDirectory() as tmp_dir:
+      self.test_target.out_dir = tmp_dir
+      self.assertTrue(
+          self.test_target.check_reproducibility_and_regression(
+              '/example/crash/testcase'))
+
+  def test_with_invalid_crash(self):
+    """Checks to make sure an invalid crash returns false."""
+    with unittest.mock.patch.object(fuzz_target.FuzzTarget,
+                                    'is_reproducible',
+                                    side_effect=[True, True]):
+      self.assertFalse(
+          self.test_target.check_reproducibility_and_regression(
+              '/example/crash/testcase'))
+
+    with unittest.mock.patch.object(fuzz_target.FuzzTarget,
+                                    'is_reproducible',
+                                    side_effect=[False, True]):
+      self.assertFalse(
+          self.test_target.check_reproducibility_and_regression(
+              '/example/crash/testcase'))
+
+      with unittest.mock.patch.object(fuzz_target.FuzzTarget,
```

Comment:
this seems to have a redundant indentation, it shouldn't be within the block started by line 124, right?

Reply:
Fixed.